requirements.txt

```diff
@@ -6,4 +6,5 @@ tabulate
 setuptools>=77.0.3,<80.0.0
 setuptools-scm>=8
 numba
-transformers>=4.1,<4.56.0
\ No newline at end of file
+transformers>=4.1,<4.56.0
+nixl==0.4.1
```

Comment:
@xuechendi The latest is 0.5.1 and it's not working for us and we use 0.5.0. Do you think we can update this to 0.5.0?

Reply:
Does 0.5.0 works for you? Will that cause the NIXL_BACKEND_ERROR?